tasks/lib/helpers/git.js

```diff
@@ -52,7 +52,7 @@ function checkout(grunt, dir, branchName, done) {
 
 function merge(grunt, dir, mergeFromBranch, msgPrefix, msg, done) {
 
-  exec('git merge --no-ff -m "' + msgPrefix + ' ' + msg + '" ' + mergeFromBranch, {
+  exec('git merge --no-ff --strategy-option theirs -m "' + msgPrefix + ' ' + msg + '" ' + mergeFromBranch, {
```

Comment:
Is this a problem for you guys? I had to use this since hotfixes otherwise cannot be merged automatically...